pycalphad/variables.py

```diff
@@ -651,6 +655,8 @@ def compute_property(self, compsets, cur_conds, chemical_potentials):
 
     def jansson_derivative(self, compsets, cur_conds, chemical_potentials, deltas: JanssonDerivativeDeltas):
         "Compute Jansson derivative with self as numerator, with the given deltas"
+        if len(compsets) == 0:
+            return np.full(self.shape, np.nan)
```

Comment:
Coverage is pinging here - any easy way to hit this code path? I thought one of the tests above would do it, but it looks like it's showing missing.

Reply:
we hit the equivalent in `JanssonDerivative.compute_property` and this method never gets called if there are no compsets. This is probably a useful guard in case anyone tries to do anything fancy later. Open to other options if we need to hit the line though.